ambari-logsearch/ambari-logsearch-server/src/main/java/org/apache/ambari/logsearch/solr/SolrConstants.java

```diff
@@ -57,6 +57,7 @@ private ServiceLogConstants() {
     public static final String LOG_MESSAGE = "log_message";
     public static final String KEY_LOG_MESSAGE = "key_log_message";
     public static final String HOST = "host";
+    public static final String GROUP = "host";
```

Comment:
Is this should be "group" instead of "host"?

Reply:
thanks